src/models/Admin.js

```diff
@@ -1,2 +1,11 @@
 import mongoose from 'mongoose'
 
+const AdminSchema = new mongoose.Schema({
+  name: { type: String, required: true },
+  // email should be unique and should be in  example@example.com formate match: /.+\@.+\..+/,
+  email: { type: String, lowercase: true, required: true, unique: true },
```

Comment:
I'm very noob in regexes, but if you can write one for proper email validation, that is simply awesome!